libia2/include/ia2.h

```diff
@@ -77,38 +77,42 @@ static uint32_t ia2_get_pkru() {
 #define STACK_SIZE (4 * 1024 * 1024)
 
 #ifdef LIBIA2_INSECURE
-#define _IA2_DEFINE_SIGNAL_HANDLER(function, pkey)    \
-    __asm__(".global ia2_sighandler_" #function "\n"  \
-            "ia2_sighandler_" #function ":\n"         \
-            "jmp " #function "\n")
+#define _IA2_DEFINE_SIGNAL_HANDLER(function, pkey)                             \
+  __asm__(".global ia2_sighandler_" #function                                  \
+          "\n"                                                                 \
+          "ia2_sighandler_" #function                                          \
+          ":\n"                                                                \
+          "jmp " #function "\n")
```

Comment:
Can we undo the formatting change here please? It's hard to see what changed.

Reply:
Or put that in a separate change on the bottom of the stack, if you really want to do it. But I prefer the old formatting.

Reply:
This is the formatting clang-format gives us, but it is pretty awful. For other macros containing asm I've been wrapping them in a `/* clang-format off */` block, so I'll do that here.